### DISCUSSION TITLE
Create new issue from task list does not use the template chooser

### DISCUSSION THREAD

**Discussion by zsloane** [community/community#72731] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Issues

### Body

When you click the "+" to add an issue to task list you have the option of creating a new issue from there if it does not already exist.

![image](https://github.com/community/community/assets/10229009/7116fef8-440b-49a6-bac5-46a7dfe45f61)

This should use the same template chooser that is presented in the "create new issue" flow from a project:
![image](https://github.com/community/community/assets/10229009/eaede082-7b3f-49ad-aab9-fb783f8a8fe6)

Otherwise this is yet another way to bypass templates when "allow blank issues" is turned off. Another being the draft->issue conversion.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by zsloane** [comment]
Any feedback on this?

**Comment by jameswberry** [comment]
+1 for this feature.

**Comment by macedigital** [comment]
I would also +1 this feature. While it is extremely helpful to be able to create issues directly from task lists, ending up with a blank issue is not great usability.

![Screenshot from 2024-03-19 19-50-53](https://github.com/community/community/assets/248495/194db721-5cee-4ab5-a9a5-0fed58ab4d74)

Also the "convert to issue" features seems to be completely bypassing `blank_issues_enabled: false` setting in `.github/ISSUE_TEMPLATE/config.yml`.
